app/views/admin/enterprise_groups/index.html.haml

```diff
@@ -17,14 +19,18 @@
     - @enterprise_groups.each do |enterprise_group|
       %tr
         %td.name= enterprise_group.name
+        - if spree_current_user.admin?
+          %td= enterprise_group.owner.andand.email || ""
         %td= enterprise_group.on_front_page ? 'Y' : 'N'
         %td= enterprise_group.enterprises.map(&:name).join ', '
         %td.actions
           = link_to '', main_app.edit_admin_enterprise_group_path(enterprise_group), class: 'edit-enterprise-group icon-edit no-text'
           = link_to_delete enterprise_group, no_text: true
 
-          - if enterprise_group.last?
-            .blank-action
-          - else
-            = link_to_with_icon 'icon-arrow-down', '', main_app.admin_enterprise_group_move_down_path(enterprise_group), class: 'move-down no-text'
-          = link_to_with_icon 'icon-arrow-up', '', main_app.admin_enterprise_group_move_up_path(enterprise_group), class: 'move-up no-text' unless enterprise_group.first?
+          - if spree_current_user.admin?
```

Comment:
Does this behave for enterprise user, where perhaps they can't see all the groups? I'm imagining them hitting the up or down buttons, and it moves it one up the global list, but doesn't affect the ordering in their filtered list.